hardhat.config.js

```diff
@@ -89,14 +100,24 @@ module.exports = {
                     browserURL: "https://gnosisscan.io/"
                 },
             },
+            {
+                network: "arbitrumSepolia",
+                chainId: 421614,
+                urls: {
+                    apiURL: "https://api-sepolia.arbiscan.io/api",
+                    browserURL: "https://sepolia.arbiscan.io"
+                },
+            },
```

Comment:
`arbitrumOne` is in the default list of hardhat-etherscan networks list, and no custom definition is needed for contract verifications.